.github/workflows/pr.yml

```diff
@@ -0,0 +1,53 @@
+name: PR
+on:
+  pull_request_target:
```

Comment:
Good question, as always :) 
I took this workflow from the [uwos-go](https://github.com/utilitywarehouse/uwos-go/blob/main/.github/workflows/pr.yml).
It makes the workflow run against the target branch of the repository, while granting all the secrets. This sort of trigger was actually added to allow secure run of workflows with secrets in the public repos. 
I suppose here it was used to make the workflow [run correctly for forks](https://stackoverflow.com/questions/74957218/what-is-the-difference-between-pull-request-and-pull-request-target-event-in-git). I don't expect many external contributions, but cool detail, I think.
The article you linked points out it might be dangerous, but
```
Generally speaking, when the PR contents are treated as passive data, i.e. not in a position of influence over the build/testing process, it is safe. But the repository owners must be extra careful not to trigger any script that may operate on PR controlled contents like in the case of `npm install`.
```

I don't mind changing it to `pull_requets`, even though it should be safe now. 
I can also just add a comment with warning to this file, to prevent dangerous contributions. 

What would you suggest?

Reply:
> I don't mind changing it to pull_requets, even though it should be safe now.

I like the idea of `pull_requests` mostly because it's used most other places (that I've seen/can remember) and also as a best-practice

Reply:
[Applied](https://github.com/utilitywarehouse/shared-kustomize-bases/pull/41/commits/0262f3fa8ab3c0367a8a4ca97d1bc719ec894307#diff-15c806aa509538190832852f439e9921a23bec2da81f95ed0e4bf13c14e5b160R3)